logos/benchmarks/benchmark_logos.py

```
@@ -3709,6 +3949,197 @@ async def _reset_and_calibrate_all_nodes(
+    re-pick that model for calibration. The benchmark needs the COMPLETE profile
+    (sleep scenarios size lanes from it), so we drop the entry to force a clean
+    recalibration. The node must be stopped first, or the running worker
+    re-saves the entry from memory. Reads/writes over SSH the same way as the
+    config helpers (cat -> edit locally -> tee); needs pyyaml on this host.
+    Returns the model names actually removed.
+    """
+    if not models:
+        return []
+    sudo = "sudo " if use_sudo else ""
+    profiles_path = f"{workernode_dir}/data/model_profiles.yml"
+    read_res = subprocess.run(
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)